v1.1/training/cluster-upgrade.md

```diff
@@ -0,0 +1,292 @@
+    ~~~ shell
+    $ ./cockroach-v2.0 start \
+    --insecure \
+    --store=node2 \
+    --host=localhost \
+    --port=26258 \
+    --http-port=8081 \
+    --join=localhost:26257,localhost:26258,localhost:26259
+    ~~~~
+
+4. Wait 1 minute.
+
+5. In node 3's terminal, press **CTRL + C** to stop the `cockroach` process.
+
+6. Verify that node 3 has stopped:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ ps | grep cockroach
+    ~~~
+
+    You should not see a `cockroach` process with `--store=node3` and `--port=26259`.
+
+    ~~~
+    42471 ttys002    0:02.01 ./cockroach start --insecure --store=node1 --host=localhost --port=26257 --http-port=8080 --join=localhost:26257,localhost:26258,localhost:26259
+    42483 ttys003    0:01.90 ./cockroach start --insecure --store=node2 --host=localhost --port=26258 --http-port=8081 --join=localhost:26257,localhost:26258,localhost:26259
+    42543 ttys004    0:00.00 grep cockroach
+    ~~~~
+
+7. Restart the node using the v2.0 binary:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ ./cockroach-v2.0 start \
+    --insecure \
+    --store=node3 \
+    --host=localhost \
+    --port=26259 \
+    --http-port=8082 \
+    --join=localhost:26257,localhost:26258,localhost:26259
+    ~~~~
+
+## Step 5. Check your cluster's versions
+
+Back in the Admin UI, you'll see that all 3 nodes now have the same, upgraded version:
+
+<img src="{{ 'images/training-21.png' | relative_url }}" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:100%" />
+
+You can also use the `cockroach node status` command to check each node's version:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ ./cockroach node status \
+--insecure
+~~~
+
+~~~
++----+-----------------+---------------------+---------------------+---------------------+
+| id |     address     |        build        |     updated_at      |     started_at      |
++----+-----------------+---------------------+---------------------+---------------------+
+|  1 | localhost:26257 | v2.0-alpha.20180212 | 2018-02-13 10:59:42 | 2018-02-13 10:47:22 |
+|  2 | localhost:26258 | v2.0-alpha.20180212 | 2018-02-13 10:59:49 | 2018-02-13 10:55:49 |
+|  3 | localhost:26259 | v2.0-alpha.20180212 | 2018-02-13 10:59:42 | 2018-02-13 10:56:12 |
++----+-----------------+---------------------+---------------------+---------------------+
+(3 rows)
+~~~
+
+## Step 6. Finalize the upgrade
+
+Once all nodes are on the same upgraded version, update the `version` cluster setting to enable certain performance improvements and bug fixes that were introduced in v2.0:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ ./cockroach sql \
+--insecure \
+--execute="SET CLUSTER SETTING version = crdb_internal.node_executable_version();"
+~~~
+
+{{site.data.alerts.callout_info}}This final step is required after upgrading from v1.1.x to v2.0. For upgrades within the 2.0.x series, you don't need to take any further action.{{site.data.alerts.end}}
```

Comment:
Shouldn't we also warn that this eliminates the ability to downgrade? We could rephrase the intro to this step as "update the `version` cluster setting to enable certain backwards-incompatible performance improvements and bug fixes that were introduced in v2.0:"